From: Mitchell Augustin <mitchell.augustin@canonical.com>
Date: Wed, 3 Jun 2026 13:39:58 +0800
Subject: Bump DEFAULT_BSS_MAX_COUNT to 1000

Bug-Ubuntu: https://bugs.launchpad.net/bugs/2117180

Many congested areas have more than 200 APs in range
of a typical device. When the number of APs exceeds
BSS_MAX_COUNT, even nearby APs can be absent from
the scanning device's AP list.
This patch bumps the default to 1000 to be more in
line with a realistic modern max number of APs
in a given area.
---
 wpa_supplicant/config.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/wpa_supplicant/config.h b/wpa_supplicant/config.h
index 8981305..c913571 100644
--- a/wpa_supplicant/config.h
+++ b/wpa_supplicant/config.h
@@ -29,7 +29,7 @@
 #define DEFAULT_P2P_INTRA_BSS 1
 #define DEFAULT_P2P_GO_MAX_INACTIVITY (5 * 60)
 #define DEFAULT_P2P_OPTIMIZE_LISTEN_CHAN 0
-#define DEFAULT_BSS_MAX_COUNT 200
+#define DEFAULT_BSS_MAX_COUNT 1000
 #define DEFAULT_BSS_EXPIRATION_AGE 180
 #define DEFAULT_BSS_EXPIRATION_SCAN_COUNT 2
 #define DEFAULT_MAX_NUM_STA 128
